Makefile.in

```diff
@@ -0,0 +1,11 @@
+PYTHON ?= /usr/local/bin/python2.7
```

Comment:
isn't 3.6 the default these days?

Reply:
There's even 3.7 coming with the latest packages. I find it okay to leave the current default (Python 2.7) as long as it can be overridden with an env variable.

Reply:
It can be passed from the command line `make install PYTHON=/usr/local/bin/python`